config_verifier/src/index.ts

```diff
@@ -0,0 +1,520 @@
+    }];
+    managerData['manager'] = manager;
+    managerData['enabledTransceiverCount'] = 1;
+
+    return managerData;
+}
+
+async function configureEvm(rpc, managerAddress){
+    const provider = new providers.JsonRpcProvider(rpc);
+
+    const managerData = {}; 
+    const signer = new ethers.VoidSigner("0x8ba1f109551bD432803012645Ac136ddd64DBA72", provider); // Actual address doesn't matter since we're a VOID signer
+    managerData['signer'] = signer; 
+
+    const Manager = await NttManager__factory.connect(managerAddress, signer);
+
+    managerData['manager'] = Manager; 
+    managerData['provider'] = provider;
+    managerData['managerAddress'] = managerAddress;
+
+    var threshold = await Manager.getThreshold();
+    var mode = await Manager.getMode();
+    var token = await Manager.token();
+    var decimals = await Manager.tokenDecimals();
+    var outboundRateLimit = (await Manager.getOutboundLimitParams())['limit']
+    var rateLimitDuration = await Manager.rateLimitDuration();
+    var chainId = await Manager.chainId();
+
+    managerData['mode'] = mode; 
+    managerData['token'] = token; 
+    managerData['threshold'] = threshold;
+    managerData['tokenDecimals'] = decimals;
+    managerData['outboundRateLimit'] = outboundRateLimit;
+    managerData['chainId'] = chainId;
+    managerData['duration'] = rateLimitDuration;
+
+    var transceivers = await Manager.getTransceivers();
+
+    // Get the creation bytecode of the contract to verify we've deployed the proper thing
+    // https://eips.ethereum.org/EIPS/eip-1967
+    // var implementationAddress = await provider.getStorageAt(managerAddress, "0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc")
+
+    // Get the transeivers in storage based upon the storage setup that we found
+    // Take a hash of the slot then add the index to find the slot of the transceiver
+    var transceiverList = [];
+    for (var i = 0; i < transceivers.length; i++){
+        var transceiverAddress = transceivers[i];
+
+        // Check to see if it's a wormhole transeiver or not
+        const transceiver = await WormholeTransceiver__factory.connect(transceiverAddress, provider);
+
+        // If it's a wormchain, then setup all of the necessary wormhole specific checks
+        var isWormhole = false;
+        try {
+
+            // If this succeeds, then it's a wormhole transeiver
+            await transceiver.getWormholePeer(1);
+            isWormhole = true;
+
+            var consistencyLevel = await transceiver.consistencyLevel();
+            var nttManager = await transceiver.nttManager();
+
+            // Only going to operate on a transceiver that we know. 
+            transceiverList.push({address: transceiverAddress, isWormhole: isWormhole, transceiver: transceiver, consistencyLevel: consistencyLevel, nttManager: nttManager});
+
+            break;
+        }
+        catch{} // defaults to not adding it
```

Comment:
Any chance you could log something here? Having a silent `try-catch` might hide some important errors